front/src/applications/stdcm/components/StdcmForm/StdcmConfig.tsx

```diff
@@ -88,10 +94,10 @@ const StdcmConfig = ({
   };
 
   const removeOriginArrivalTime = () => {
-    dispatch(updateOriginArrivalType(ArrivalTimeTypes.ASAP));
+    dispatch(updateStdcmPathStep({ ...origin, arrivalType: ArrivalTimeTypes.ASAP }));
```

Comment:
Hm, this seems racy when two dispatches happen in quick succession… Right?

Could we maybe do the merging with the current state inside the action?